README.md

```diff
@@ -145,6 +145,10 @@ To run the tests:
 
     pytest
 
+To run the `festerize` executable for manual testing:
+
+    ./venv_festerize/bin/festerize --help
+
```

Comment:
I'm not sure what the purpose of this is. If the `venv_festerize` virtual environment is activated, e.g. with
```
. venv_festerize/bin/activate
```
then `./venv_festerize/bin/festerize` will already be on the user's PATH

Reply:
Doh! Okay, thanks.